R/class-AbundanceDensityPlot.R

```diff
@@ -164,6 +178,8 @@ setMethod(".generateOutput", "AbundanceDensityPlot",
     args[["layout"]] <- deparse(slot(x, "layout"))
     args[["add_legend"]] <- deparse(slot(x, "add_legend"))
     args[["assay.type"]] <- deparse(slot(x, "assay.type"))
+    args[["flipped"]] <- deparse(slot(x , "flipped"))
+    args[["order_descending"]] <- deparse(slot(x, "order_descending"))
```

Comment:
First, descending = FALSE and NA both give the same order. Could you check if this is the case only for Tengeler2020 or for also for other datasets? Or maybe it's explained in the function vignettes?

If FALSE and NA always give the same order, you can just reduce this parameter to a checkbox with TRUE and FALSE.

Reply:
if NA, it chooses either TRUE or FALSE based on the value found in the TSE object (at least this is what documentation says)

Reply:
Ok then TRUE/FALSE checkbox is probably enough